source/macros.tex

```diff
@@ -198,7 +198,7 @@
 \newcommand{\leftshift}[1]{\ensuremath{\, \mathsf{lshift}_#1 \,}}
 
 %% Notes and examples
-\newcommand{\noteintro}[1]{[\,\textit{#1:}\space}
+\newcommand{\noteintro}[1]{[\,\textit{#1}:\space}
```

Comment:
I've also always wanted to reduce the excessive space after the `[` and before the `]`. Instead of `\,`, I figured something like a `\kern0.25pt` or so would suffice. But I'll need to convince @zygoloid  of that first.